app/controllers/users.js

```diff
@@ -41,3 +42,24 @@ exports.getUsers = async (req, res, next) => {
     next(error);
   }
 };
+exports.adminLog = async (req, res, next) => {
+  try {
+    const userObject = req.body;
+    const userFound = await userService.findUser(userObject);
+    if (userFound === null) {
```

Comment:
```suggestion
    if (!userFound) {
```

Reply:
There is a problem with that negated if, lint doesn't allow negeted conditions

Reply:
Try putting whit out parenthesis
```suggestion
    if !userFound {
```

Reply:
it expect a parenthesis

Reply:
ok ok, no problem